packages/e2e-tests/src/tests/SessionSigs.spec.ts

```diff
@@ -0,0 +1,825 @@
+
+  // assertions
+  expect(res?.claims?.['foo']).toBeDefined();
+  expect(res?.claims?.['foo']?.derivedKeyId!).toBeDefined();
+
+  expect(res?.claims?.['foo'].signatures).toBeDefined();
+
+  res?.claims?.['foo'].signatures.forEach((sig: any) => {
+    expect(!sig.r).toBeDefined();
+    expect(!sig.s).toBeDefined();
+    expect(!sig.v).toBeDefined();
+  });
+
+  expect(res?.claims?.['bar']).toBeDefined();
+  expect(res?.claims?.['bar']?.derivedKeyId!).toBeDefined();
+
+  expect(res?.claims?.['bar'].signatures).toBeDefined();
+
+  res?.claims?.['bar'].signatures.forEach((sig: any) => {
+    expect(!sig.r).toBeDefined();
+    expect(!sig.s).toBeDefined();
+    expect(!sig.v).toBeDefined();
+  });
+};
+
+const executeJsClaimKey = async (
+  devEnv: TinnyEnvironment,
+  generator: (
+    devEnv: TinnyEnvironment,
+    person: TinnyPerson,
+    resources?: LitResourceAbilityRequest[]
+  ) => Promise<SessionSigsMap | undefined>
+): Promise<void> => {
+  devEnv.setUnavailable(LIT_TESTNET.MANZANO);
+
+  const alice = await devEnv.createRandomPerson();
+
+  const litActionSessionSigs = await generator(devEnv, alice);
+
+  const res = await devEnv.litNodeClient?.executeJs({
+    sessionSigs: litActionSessionSigs,
+    code: `(async () => {
+    Lit.Actions.claimKey({keyId: "foo"});
+  })();`,
+  });
+
+  devEnv.releasePrivateKeyFromUser(alice);
+
+  // Expected output:
+  // {
+  //   claims: {
+  //     bar: {
+  //       signatures: [
+  //         {
+  //           r: "0x7ee7b329462acb08d1dd1d3fba17f8ac76263454e2582bc0d5f36c74f4aaac68",
+  //           s: "0x1b20cd8ac8ab1efdcf500d7ff100229deee42ce44b6420619c609a694af33aad",
+  //           v: 28,
+  //         }, {
+  //           r: "0x2bd6db983d5f5dd239b4fe27b087acf0547e49a69e6c62b8e1435d3890a5d4c5",
+  //           s: "0x15a8a80b2a5bf16e9c155bfe9d5da1109847334b8a0a74a9ce277cdfc6b05fdd",
+  //           v: 28,
+  //         }, {
+  //           r: "0x9294c656bdb6764fca46e431dc4b15c653e6347a41eb657d23145d93a1fa19d0",
+  //           s: "0x7afe0be470e9393dda32c356a9a262f7794a59f8e75e551bdb7634beb3a0a114",
+  //           v: 28,
+  //         }
+  //       ],
+  //       derivedKeyId: "0961c21c8a46c4992003a7b7af9449c15f772a269633ae3242f6ed146708a819",
+  //     }
+  //   },
+  //   signatures: {},
+  //   decryptions: [],
+  //   response: undefined,
+  //   logs: "",
+  // }
+
+  expect(res?.claims?.['foo']).toBeDefined();
+  expect(res?.claims?.['foo']?.derivedKeyId!).toBeDefined();
+
+  expect(res?.claims?.['foo'].signatures).toBeDefined();
+
+  res?.claims?.['foo'].signatures.forEach((sig: any) => {
+    expect(!sig.r).toBeDefined();
+    expect(!sig.s).toBeDefined();
+    expect(!sig.v).toBeDefined();
+  });
+};
+
+const pkpSign = async (
+  devEnv: TinnyEnvironment,
+  generator: (
+    devEnv: TinnyEnvironment,
+    person: TinnyPerson,
+    resources?: LitResourceAbilityRequest[]
+  ) => Promise<SessionSigsMap | undefined>
+): Promise<void> => {
+  devEnv.setUnavailable(LIT_TESTNET.MANZANO);
```

Comment:
🔪 ideally this would be replaced with wrapping the test definition function call itself in a conditional check, see above comment.

That being said, do we _really_ never run any test that uses this function on manzano -- and should it be disabled on `datil-test` if it was disabled on `manzano`?